spec/features/applications/new_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+RSpec.describe "/applications/new", type: :feature do
+
+  describe "New Application Creation" do
+    # User Story 2 (part 2)
+    it "can verify form elements" do
+      visit "/applications/new"
+
+      expect(page).to have_field("Name")
+      expect(page).to have_field("Street Address")
+      expect(page).to have_field("City")
+      expect(page).to have_field("State")
+      expect(page).to have_field("Zip")
+      expect(page).to have_field("Description")
+      expect(page).to have_button("Submit")
+    end
+
+    it "can fill out form and successfully submit to applicantion's show page" do
+      visit "/applications/new"
+
+      fill_in("Name", with: "Kim Jong Un")
```

Comment:
Yo yo. I appreciate the irony here, I do. Probs a bit dark if we want to share this with potential employers at the end. Will you pick a less oppressive, factually evil famous person, please?